paths/account_locales/index.yaml

```diff
@@ -0,0 +1,42 @@
+---
+summary: List locales used in account
+description: List all locales used across all projects within an account.
+operationId: account/locales
```

Comment:
@forelabs i didn't notice the file path here, but i think this should go in ` paths/accounts/locales.yaml`

Reply:
I had this first but was not sure, I thought I found something else which had also a new folder though it was part of a different path. But yeah from the path point of view it makes more sense. I can change that, no hard feelings about that.